web/src/components/dashboard/QuickActionTile.tsx

```diff
@@ -14,7 +14,12 @@ interface TaskProps {
   quickAction: QuickActionTask;
 }
 
-export const QuickActionTile = (props: LinkProps | TaskProps): ReactElement => {
+interface LicenseReinstatementProps {
+  type: "license";
```

Comment:
[pebble] why does this need to be a prop if it's always going be of license type, do you see it changing in the future to include more types?

Reply:
@cbrunefearless @somabadri Initially, I was thinking that this wasn't necessary. But looking at the [next story we're going to tackle](https://www.pivotaltracker.com/story/show/186880077) related to the reinstatement quick action, we're going to be building in functionality based on a given license type being expired. While these are quick actions that will drive to tasks, we only want them to show up conditionally based on the related license being expired. We'll want to be able to check that a given quick action is of type `license` in order to conditionally show it.